ROADMAP.md

```diff
@@ -0,0 +1,111 @@
+# Roadmap
+
+Last updated: 2024-03-12
+
+## Background
+
+This document sets out the plan for the evolution of the Data Safe Haven codebase.
+It covers both short term-goals, which are organised in sprint milestones, and long-term goals, which are expected without a particular time frame.
+It also collects desirable features which are unplanned or which we have decided not to pursue yet.
+
+The [short-term plans](#short-term) aim to give a clear indication of current work and when it may be finished.
+We hope that a focus on defining releases will encourage development to balance new features with other improvements and address the needs of users.
+
+[Long-term goals](#long-term) give reassurance that we have committed to certain changes when we are unable to give a estimation of when they will be ready.
+
+Finally, [desirable features](#desireable-features) is a way for us to express developments we would like to pursue but have decided not to do yet.
+This is a good indication of where we would like the project to go and areas we would be interested in collaborating.
+
+## Governance
+
+This document is written by the code's [development lead](https://github.com/JimMadge) with input from the development team.
+
+When deciding on what features to include in the roadmap, and where to include them we consider, in no particular order,
+
+- The maintainability of the code (looking after the developers)
+- Supporting users at The Alan Turing Institute (a significant user who we have a close relationship with)
+- Supporting all users we are in contact with
```

Comment:
I think that is kind of implicit here, we decide those things _don't_ go on the roadmap.

It would be good to add a note like that though if we had a system for addons/plugins. As we don't now, it feels a bit misleading.

Reply:
Mb that system could go in desired features?

Reply:
That's a great idea actually. I've brought up addons/plugins a few times so I don't know why I forgot it :smile:.

Reply:
Done in 4141fc7